plugins/search-react/api-report.md

```diff
@@ -314,55 +320,50 @@ export const SearchResultGroupFilterFieldLayout: (
 // @public
 export type SearchResultGroupFilterFieldLayoutProps = PropsWithChildren<{
   label: string;
-  value?: JsonValue;
+  value?: string;
   onDelete: () => void;
 }>;
 
 // @public
 export type SearchResultGroupFilterFieldPropsWith<T> = T &
   SearchResultGroupFilterFieldLayoutProps & {
-    onChange: (value: JsonValue) => void;
+    onChange: (e: string) => void;
   };
 
 // @public
-export function SearchResultGroupLayout<FilterOption>(
-  props: SearchResultGroupLayoutProps<FilterOption>,
-): React_2.JSX.Element;
+export function SearchResultGroupLayout<T>(
+  props: SearchResultGroupLayoutProps<T>,
+): JSX.Element | null;
 
 // @public
-export type SearchResultGroupLayoutProps<FilterOption> = ListProps & {
+export type SearchResultGroupLayoutProps<T> = ListProps & {
   error?: Error;
   loading?: boolean;
   icon: JSX.Element;
   title: ReactNode;
   titleProps?: Partial<TypographyProps>;
   link?: ReactNode;
   linkProps?: Partial<LinkProps>;
-  filterOptions?: FilterOption[];
-  renderFilterOption?: (
-    value: FilterOption,
-    index: number,
-    array: FilterOption[],
-  ) => JSX.Element | null;
+  filterOptions?: T[];
+  renderFilterOption?: T extends string | number | undefined | null
+    ? RenderFilterOption<T> | undefined
+    : RenderFilterOption<T>;
   filterFields?: string[];
   renderFilterField?: (key: string) => JSX.Element | null;
   resultItems?: SearchResult_2[];
   renderResultItem?: (
     value: SearchResult_2,
-    index: number,
-    array: SearchResult_2[],
+    index?: number,
+    array?: SearchResult_2[],
```

Comment:
Making these optional is a breaking change, since the callback switches this from an input to output type